tests/testthat/test-codelist.R

```diff
@@ -84,7 +84,23 @@ test_that("create_var_from_codelist", {
     select(PARAMCD, PARAM) |>
     expect_equal(compare)
 
-  # Test warning where arg `strict == TRUE`
+  # Test character variable warning where arg `strict == TRUE` / single issue case
+  create_var_from_codelist(
+    data = data,
+    metacore = adlb_spec,
+    input_var = PARAMCD,
+    out_var = PARAM,
+    codelist = get_control_term(adlb_spec, PARAMCD),
```

Comment:
```suggestion
    codelist = metacore::get_control_term(adlb_spec, PARAMCD),
```

I thought this was from metatools at first and got confused...maybe be more explicit here with the `::` for those who come later?